lib/MilkCheck/ServiceManager.py

```diff
@@ -148,14 +151,15 @@ def call_services(self, services, action, conf=None):
         self.reset()
         self.variables.clear()
 
-        # Create global variable from configuration
-        self._variable_config(conf)
         if conf:
             # Apply configuration over the graph
             self._apply_config(conf)
             # Enable reverse mode if needed, based on config
             self.algo_reversed = action in conf.get('reverse_actions')
 
+        # Create global variable from configuration
+        self._variable_config(conf)
```

Comment:
I'm remember this position in the code being special. I'm also surprised everything is fine.
Are you sure there are no side effects?

By example, it looks like excluded_nodes will never be set now that this method is called later in the code

Reply:
Tests are fine and i tested it manually and nodes are correctly excluded. Same for services.

Reply:
Ok, let's say it was some time ago when the code was different and now it is fine... :/
You will see soon :)